module/vision/BallDetector/src/BallDetector.cpp

```diff
@@ -271,12 +275,15 @@ namespace module::vision {
                     }
 
                     // Discard this ball if projection_distance and distance are too far apart
-                    if ((std::abs(projection_distance - distance) / max_distance) > config.distance_disagreement) {
+                    const float max_distance = std::max(projection_distance, angular_distance);
+
+                    if ((std::abs(projection_distance - angular_distance) / max_distance)
+                        > config.distance_disagreement) {
```

Comment:
@TrentHouliston do we even need this throwout if we are passing both distance measurements to localisation? Wouldnt it be better to let localisation deal with the discrepency?

Reply:
I assumed this was a throw out because it implies an assumption made didn't really work out, eg ball radius, and so it's probably not a ball. When I was tuning the other day with a new mesh, this was a useful throwout to get rid of some bad detections.

Reply:
Localisation could do more with it, but in this case it's when it gets to the "this isn't a ball is it" threshold. Like if it saw a ball on a poster on the wall or something